src/compas_tna/diagrams/formdiagram.py

```diff
@@ -173,6 +186,326 @@ def from_mesh(cls, mesh: Mesh) -> "FormDiagram":
         """
         return mesh.copy(cls=cls)
 
+    # --------------------------------------------------------------------------
+    # create form diagrams from templates
+    # --------------------------------------------------------------------------
+
+    @classmethod
+    def create_cross_form(cls, xy_span=[[0.0, 10.0], [0.0, 10.0]], discretisation=10, supports="corners") -> "FormDiagram":
+        """Construct a FormDiagram based on cross discretisation with orthogonal arrangement and diagonal.
+
+        Parameters
+        ----------
+        xy_span : list, optional
+            List with initial- and end-points of the vault, by default [[0.0, 10.0], [0.0, 10.0]]
+        discretisation : int, optional
+            Set the density of the grid in x and y directions, by default 10
+        supports : str, optional
+            Option to select the constrained nodes: 'corners', 'all' are accepted., by default 'corners'
+
+        Returns
+        -------
+        :class:`~compas_tno.diagrams.FormDiagram`
+            The FormDiagram created.
+
+        """
+        mesh = create_cross_form(xy_span=xy_span, discretisation=discretisation)
+        form = cls.from_mesh(mesh)
+        form.assign_support_type(supports)
+
+        return form
+
+    @classmethod
+    def create_fan_form(cls, xy_span=[[0.0, 10.0], [0.0, 10.0]], discretisation=10, supports="corners") -> "FormDiagram":
```

Comment:
`xy_span` as list of list of list is confusing. `[[x0, y0], [x1, y1]]` or `[[x0, x1], [y0, y1]]`?

why not just do `x_span` and `y_span`?